src/styles/SimulationResultsStyles.ts

```diff
@@ -0,0 +1,104 @@
+import styled from "styled-components";
+import { ButtonColors, DarkTheme } from "./MainStyles";
+
+export const SimulationResultsUl = styled.ul`
+	list-style-type: none;
+	display: flex;
+	flex-direction: column;
+	flex-wrap: no-wrap;
+	gap: 2vh;
+	justify-content: flex-start;
+	align-items: flex-start;	
+	width: 90%
+	height: 80vh;
+	padding: 1vh 1vw;
+`;
+
+
+export type CHProps = { topMargin: number; leftMargin: number;}
+
+export const CustomHeader = styled.p<CHProps>` 
+	font-weight: bold; 
+	margin: ${(props) => props.topMargin}px 5px 15px ${(props) => props.leftMargin}px;
+`;
+
+export type CPProps = { topMargin: number; }
+
+export const CustomParagraph = styled.p<CPProps>` 
+	font-weight: normal; 
+	margin: ${(props) => props.topMargin}px 15px 15px 0px;
+`;
+
+export const SimulationResultsLi = styled.li`
+	display: flex;
+	flex-direction: column;
+	flex-wrap: no-wrap;
+	justify-content: flex-start;
+	align-items: flex-start;
+	
+	//padding: 1vh 1dvw;
+	width: calc(100% - 0.5vw);
+	background-color: ${(props) => props.theme.secondary};
+	border-radius: 15px;
+	border: solid ${(props) => props.theme.text} 1px;
+`;
+
+export const SimulationResultsPanel = styled.div`
```

Comment:
od `SimulationResultsLi` cały kod jest dosłownie przekopiowany z `NumericResultsStyles.ts`, po coś te exporty są... Jeśli nie pasują ci nazwy to je zunifikuj, np. do `ResultsLi`, ale nie kopiuj ich jak chcesz tylko nazwę zmienić błagam cię!!!